app/src/main/java/de/danoeh/antennapod/activity/PreferenceActivity.java

```diff
@@ -159,4 +171,28 @@ public void onSearchResultClicked(SearchPreferenceResult result) {
             result.highlight(fragment);
         }
     }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+
+        try {
+            AccountImporter.onActivityResult(requestCode, resultCode, data, this, account -> {
+                // As this library supports multiple accounts we created some helper methods if you only want to use one.
+                // The following line stores the selected account as the "default" account which can be queried by using
+                // the SingleAccountHelper.getCurrentSingleSignOnAccount(context) method
+                SingleAccountHelper.setCurrentAccount(getApplicationContext(), account.name);
+            });
+        } catch (AccountImportCancelledException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Override
+    public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
```

Comment:
Is this really needed? Is it used for getting the email address?

Reply:
this is not being called. I assume its a copy&paste artifact. removed it.